players_card_sprites_area.py

```diff
@@ -0,0 +1,26 @@
+import arcade
+
+from screen_configuration import ScreenConfiguration
+
+
+class PlayersCardSpritesArea:
+    def __init__(self, pile_mat_list, config: ScreenConfiguration):
+        self.config = config
+        self.pile_mat_list = pile_mat_list
+        self.start_x_position = self.config.start_x + self.config.x_spacing
+        self.__init_with_twelve_sprites()
+
+    def __init_with_twelve_sprites(self):
```

Comment:
Generally looks really nice. But I wouldn't place all twelve mats in this class and rather have an extra class for the computer player.